src/main/java/homeworks/christian/firstAssigment/secondAssigment/InterfaceA.java

```diff
@@ -0,0 +1,12 @@
+package homeworks.christian.firstAssigment.secondAssigment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public interface InterfaceA {
```

Comment:
It should be a parent class so it can store the generic attributes and methods from those who inherit